pom.xml

```diff
@@ -1129,6 +1129,10 @@
                   <property name="format" value="&amp;quot; [+] &amp;quot;" />
                   <property name="message" value="Unnecessary concatenation of string literals" />
                 </module>
+                <module name="RegexpSinglelineJava">
+                  <property name="format" value="com.google.common.cache[.]" />
```

Comment:
The pattern should be: `"com[.]google[.]common[.]cache[.]"

Reply:
Good catch